# Lidar CV
# StratoMiners

lidar_cv:
  point_cloud_topic: /rear_velodyne/points
  human_points_topic: /lidar_cv/humanPoints
  clouds_with_confidence: /lidar_cv/lidarCoudsWithConfidence
  transform_frame: rear_velodyne_base_link
  working_frame: base_footprint
  expected_human_density: 0.15